Orange/classification/_tree_scorers.pyx

```diff
@@ -0,0 +1,293 @@
+#cython: embedsignature=True
```

Comment:
Also `boundscheck=False, wraparound=False` (ensure they can be applied module-wide). If you can vouch never to divide by zero, also `cdivision=True`.